demisto_sdk/commands/common/update_id_set.py

```diff
@@ -679,13 +679,19 @@ def get_fields_by_script_argument(task):
                 # the value should be a list of dicts in str format
                 custom_field_value = list(field_value.values())[0]
                 if isinstance(custom_field_value, str):
-                    custom_fields_list = json.loads(custom_field_value)
-                    if not isinstance(custom_fields_list, list):
-                        custom_fields_list = [custom_fields_list]
-                    for custom_field in custom_fields_list:
-                        for field_name in custom_field.keys():
-                            if field_name not in BUILT_IN_FIELDS:
-                                dependent_incident_fields.add(field_name)
+                    if custom_field_value.startswith("$"):
+                        logger.info("You're using an unrecommended method - ${} - to retrieve values from the context data.")
```

Comment:
```suggestion
                        logger.warning("You're using an unrecommended method - ${} - to retrieve values from the context data.")
```
I would make it warning

Reply:
Fixed.